common-files/test/utils.test.mjs

```diff
@@ -119,8 +120,24 @@ describe('Utils tests', function () {
     assert.equal(result, '');
   });
 
-  it("Shouldn't obfuscate for the Dev environment", () => {
-    process.env.NODE_ENV = 'developer';
+  it("Shouldn't obfuscate for the local environment", () => {
+    process.env.ENVIRONMENT = 'local';
```

Comment:
This test was removed in this [PR](https://github.com/EYBlockchain/nightfall_3/pull/1088) since all sensitive data will be obfuscated whatever the environment.